### ISSUE TITLE
Add Cognitive Services Text Sentiment on Comments

### ISSUE THREAD

**Issue by jwood803** [csharpfritz/CoreWiki#281]
Would be interesting to add some AI such as text sentiment on comments. Not exactly sure how to get/use the API key, though, but I do believe it has a free tier when creating it in Azure.

**Comment by csharpfritz** [comment]
We may work on this at Ignite 2018

**Comment by fboucher** [comment]
So based on the discussion on the show:

- We could add a Sentiment/ Happiness emoji on the side of the comment
- We could add a new pages Statistics to see globally the Sentiment results
- We must put the sentiment in some kind of *optional* module
- It should be a post process or async process (ex: in an Azure Function)

**Comment by jwood803** [comment]
For the API key, I'm guessing it's just best to have it reside in the config file?

**Comment by fboucher** [comment]
Yes, en that could be our check to activate or not all the features.

@jwood803 you mentioned you had started something. How can I help? 
You mention having something started already. Or I can get started on a different branch.

**Comment by jwood803** [comment]
@FBoucher [Here's the branch](https://github.com/jwood803/CoreWiki/tree/comment-sentiment) that I was working on. You'll notice that there's not much there, though. 😞

**Comment by fboucher** [comment]
No Worries.
Jeff added a new branch *project_Azure* with a new project in it.
We will be able to place the code there and leverage the First_Start feature.
